server/task_executor_pitr_cutover.go

```diff
@@ -85,6 +86,18 @@ func (*PITRCutoverTaskExecutor) GetProgress() api.Progress {
 	return api.Progress{}
 }
 
+func retry(maxRetry int, f func() error) error {
+	var errList error
+	for i := 0; i < maxRetry; i++ {
+		if err := f(); err != nil {
+			errList = multierr.Append(errList, errors.Wrap(err, "failed to do cutover for PostgreSQL"))
```

Comment:
this method doesn't indicate postgres, and the log seems to duplicate with the caller log

Reply:
Also it would be ideal to emit debug level retry log